source/content/guides/wordpress-composer/02-create-new-basic-wp-composer-site.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Create a Basic WordPress Site with Composer on Pantheon
+subtitle: Create a Basic Composer-managed WordPress Site 
+description: Learn how to create a basic Composer-managed WordPress site on Pantheon.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/create-new-basic-wp-composer-site
+anchorid: create-new-basic-wp-composer-site
+---
+
+This section provides steps for creating a [Composer](/guides/composer)-managed WordPress site on Pantheon. This process creates a new WordPress site with a basic setup. Visit the [Create an Advanced Composer-managed WordPress Site](/guides/wordpress-composer/create-wp-site-composer-ci-auto-test) section of this guide if you want to create a Composer-managed site that uses Continuous Integration and automated testing.
+
+ You can review the Pantheon-maintained [WordPress Composer Repo](https://github.com/pantheon-systems/wordpress-composer/) for more details on the basic Composer-managed WordPress site creation process.
```

Comment:
I think that most users would be confused if they examined the wordpress-composer repository. In any event, I think that this page should be removed until we have a supported (Bedrock-based) Composer-managed WordPress upstream.